sdcm/cluster.py

```diff
@@ -4370,7 +4381,7 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
             node.install_package('python3')
 
         node.update_repo_cache()
-
+        node.install_package('lsof')
```

Comment:
we'll need to test it across all distros (if the package is named the same)

Reply:
we install it, if it's not found

Reply:
and yes, I think it was added to the image at some point

Reply:
from all sources I found over internet, lsof and netstat are coming from the same packages: `lsof` and `net-tools`